tests/lib/rules/one-var.js

```diff
@@ -581,6 +646,7 @@ ruleTester.run("one-var", rule, {
         },
         {
             code: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3, baz = z; for (var d in baz) {}}",
+            output: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3; var baz = z; for (var d in baz) {}}",
```

Comment:
Can you add a test where a variable declaration spans multiple lines?

For example, I would expect the following code

```js
var foo = 1,
    bar = 2;
```

...to get fixed to something like this:

```js
var foo = 1;
var bar = 2;
```

It would also be fine for it to get fixed to something like this, because the `indent` rule would adjust it afterwards:

```js
var foo = 1;
    var bar = 2;
```

Reply:
Here’s what I have now:

```js
var foo = 1;
var
    bar = 2;
```

Reply:
I think if possible, this should be changed to the output that @not-an-aardvark suggested. We have no other rule that would be able to fix extra line break for the second declaration.